src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -1,13 +1,25 @@
 package com.minecolonies.coremod.event;
 
 import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.blockout.Log;
 import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.blocks.BlockConstructionTape;
+import com.minecolonies.coremod.blocks.BlockConstructionTapeCorner;
+import com.minecolonies.coremod.blocks.ModBlocks;
 import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.items.*;
 import com.minecolonies.coremod.network.messages.ColonyStylesMessage;
 import com.minecolonies.coremod.network.messages.ServerUUIDMessage;
+import com.minecolonies.coremod.sounds.ModSoundEvents;
+import net.minecraft.block.Block;
+import net.minecraft.client.audio.Sound;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.audio.Sound'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)